po/vi.po

```diff
@@ -3267,6 +3270,10 @@ msgid ""
 "`loop`, this can take an optional expression that becomes the value of the "
 "`loop` expression."
 msgstr ""
+"Nếu học viên muốn kết thúc một vòng lặp sớm, sử dụng [`break`](https://doc."
```

Comment:
Where does this break part come from? I thought the sentences only contain the two following setences:
"`loop`, this can take an optional expression that becomes the value of the "
"`loop` expression."